pkgs/by-name/dm/dmarc-report-converter/package.nix

```diff
@@ -0,0 +1,29 @@
+{ buildGoModule, fetchFromGitHub, lib, pkgs }:
```

Comment:
```suggestion
{ lib
, buildGoModule
, dmarc-report-converter
, fetchFromGitHub
, runCommand
}:
```
this is nicer to read also please don't use pkgs in this scope, rather call the package directly.

Reply:
Is it a convention that lib is at the beginning?
I ordered it alphabetically for the moment.

Reply:
Yes lib is usually the first input and then alphabetically. I think we do this because you basically always need lib, but don't remember.

Reply:
Corrected.